### ISSUE TITLE
about the output file

### ISSUE THREAD

**Issue by morningphase** [gloooryyt/oddball_py3#2]
I'm wondering what's the meaning of  the values in output file. for example, what's the meaning of "3 24.363742089079686", mayber "3" is the node id, "24.363742089079686" means the anomaly score?

**Comment by gloooryyt** [comment]
> I'm wondering what's the meaning of the values in output file. for example, what's the meaning of "3 24.363742089079686", mayber "3" is the node id, "24.363742089079686" means the anomaly score?

Yes. Node with bigger anomaly score means that it's more anomalous in the network.
